crates/sui-json-rpc/src/api.rs

```diff
@@ -84,18 +87,18 @@ pub trait RpcReadApi {
     async fn get_transactions_in_range(
         &self,
         /// the matching transactions' sequence number will be greater than or equals to the starting sequence number
-        start: GatewayTxSeqNumber,
+        start: TxSequenceNumber,
         /// the matching transactions' sequence number will be less than the ending sequence number
-        end: GatewayTxSeqNumber,
-    ) -> RpcResult<Vec<(GatewayTxSeqNumber, TransactionDigest)>>;
+        end: TxSequenceNumber,
+    ) -> RpcResult<Vec<TransactionDigest>>;
```

Comment:
Similar to my comment for `get_recent_transactions`, can we remove this?

Reply:
`get_transaction_in_range` will need some work in Explore to migrate to the new method, the old pagination uses Seq Number to get the next page but the new pagination will require storing the cursor and use it for next page, I am not familiar with typescript and the explorer codebase so might not be the best person to make this change.